python/lsst/ctrl/execute/slurmPlugin.py

```diff
@@ -214,6 +214,16 @@ def loadSlurm(self, name):
         else:
             self.defaults["RESERVATION"] = ""
 
+        if self.opts.exclude is not None:
+            self.defaults["EXCLUDE"] = f"#SBATCH --exclude {self.opts.exclude}"
+        else:
+            self.defaults["EXCLUDE"] = ""
+
+        if self.opts.nodelist is not None:
+            self.defaults["NODELIST"] = f"#SBATCH --nodelist {self.opts.nodelist}"
+        else:
+            self.defaults["NODELIST"] = ""
+
```

Comment:
Why allow both exclude and nodelist to be specified at same time?   The jobs must land on a node in the nodelist (i.e., requirement not a preference), right?  Is exclude ignored?  Is it applied to the nodelist (i.e., a different way to remove a node from the nodelist than just manually removing it from the command line list)?

Reply:
These are mostly just offering the "pass through", so it is sort of left up to the user to provide sensible inputs. 
However I can think of a hypothetical scenario that justifies this freedom, in that you might want to run on any of a range  sdfmilan1[00-99] but not on the bad node sdfmilan134 .  I tested 
 --nodelist sdfmilan0[01-30] --exclude sdfmilan005
 and it worked (I obtained glideins on sdfmilan014 sdfmilan027 .
 Making it more extreme:
 --nodelist sdfmilan0[32-33] --exclude sdfmilan033 
 gave me four glideins all on sdfmilan032, so I think it works (excludes applied to the nodelist).